index.html

```diff
@@ -333,14 +338,14 @@ <h2>NEL policies</h2>
 
       <p>
       A <a>NEL policy</a> is <dfn>stale</dfn> if the <a>duration from</a> its
-      <a>creation</a> to the <a>current wall time</a> is greater than 172800
-      seconds (48 hours).
+      <a>creation</a> to the [=environment settings object/current wall time=]

```

Comment:
I'm not sure that a NEL policy has an associated settings object - we might want something like https://w3c.github.io/hr-time/#wall-clock-unsafe-current-time here instead.

Reply:
Yeah - good point. I guess this was a bug before but less obvious (or maybe not, don't fact check me on that).

Reply:
OK, it does seem like the "wall clock unsafe current time" is what we want vs the "monotonic clock's unsafe current time", based on what Chromium using [`base::Time`](https://source.chromium.org/chromium/chromium/src/+/main:base/time/time.h;l=517-520;drc=2176b043b513af28108c91e30d38d0ed1f0dd777) in its NEL implementation to consider max age and staleness, e.g. https://source.chromium.org/chromium/chromium/src/+/main:net/network_error_logging/network_error_logging_service.cc;l=637.